tests/test_travis_multiplePS/check_correctness_cirrus_sgd.cpp

```diff
@@ -0,0 +1,89 @@
+#include <unistd.h>
```

Comment:
Change the name of the file to something consistent with the other travis tests we have.

Reply:
Fixed